src/Widgets/PricingWidget/PricingWidgetEditingConfig.js

```diff
@@ -67,4 +67,90 @@ Scrivito.provideEditingConfig("PricingWidget", {
       url: "https://scrivito.com",
     }),
   },
+  validations: [
+    [
+      "currency",
+
+      currency => {
+        if (!currency) {
+          return {
+            message: "The currency should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+    [
+      "smallPlanPrice",
+
+      smallPlanPrice => {
+        if (!smallPlanPrice) {
+          return {
+            message: "The small plan price should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+    [
+      "mediumPlanPrice",
+
+      mediumPlanPrice => {
+        if (!mediumPlanPrice) {
+          return {
+            message: "The medium plan price should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+    [
+      "largePlanPrice",
+
+      largePlanPrice => {
+        if (!largePlanPrice) {
+          return {
+            message: "The large plan price should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+    [
+      "smallPlanButton",
+
+      smallPlanButton => {
+        if (!smallPlanButton) {
+          return {
+            message: "The URL for small plan's button should be set.",
```

Comment:
One can actually also link an obj. Let's go with target instead. Maybe with:

`The small plan's button target should be set.`